### DISCUSSION TITLE
New `status: RFC` for pull requests

### DISCUSSION THREAD

**Discussion by mattpap** [bokeh/bokeh#12708] [Governance]
Currently there is no middle ground between `status: WIP` and `status: ready`, and it seems to me that the general consensus is not to pay too much attention to WIP PRs , which is understandable as they can be in various states of not working or being unviable, and wait for them to be fully ready. However, this is counterproductive, because there is a stage in the life of a PR, when it's still work-in-progress (e.g. is missing sufficient test coverage), but it good enough for discussing API changes, etc. Thus I propose to fill in the gap with `status: RFC` (Ready for Comments).

**Comment by bryevdv** [comment]
FWIW "Ready" was originally conceived as "ready for review". And this pre-dates the existence of official GH reviews, BTW. I would actually purpose something different: start using "ready" as it was intended, and adopt a self-merge policy (with a formal review approval or lapsed statement of intent to merge) [1]. In that case a signifier for "ready to merge" is superfluous and unnecessary. 

[1] There is a WIP draft here: https://github.com/bokeh/bokeh/wiki/BEP-11:-Merge-Policy

**Comment by ianthomas23** [comment]
I've checked BEP-1 (https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management) which I should know but don't fully yet and it does indeed state that `status: ready` means ready for review. So we could just follow that more closely.
